### ISSUE TITLE
Grafana v4.6.2 not work Asynchronously 

### ISSUE THREAD

**Issue by iShift** [grafana/grafana#10205]
Read before posting: 

Please include this information:
- What Grafana version are you using? **Grafana v4.6.2 (commit: 8db5f08)**
- What datasource are you using? **Clickhouse-Vertamedia**
- What OS are you running grafana on? **Ubuntu**
- What did you do? **Try to work with my dashboard**
- What was the expected result?  **# Asynchronous graphs loading**
- What happened instead? **Grafana # wait all query to complete and after that - show graphs**

tldr; I use old grafana  Grafana v4.4.1 (commit: 6a9f8ca) and all my graphs loads async, after update to last stable Grafana v4.6.2 (commit: 8db5f08) - i have lag when loading same graphs - it loads synchronously.

**Comment by bergquist** [comment]
This seems weird. Should not behave that way. 

If you open the developers tool in your browser and refresh the dashboard. Do are the requests going out synchronously?

**Comment by iShift** [comment]
No, it work async 
![image](https://user-images.githubusercontent.com/426427/33983801-86bf4962-e0c6-11e7-86ec-ff974df33e4a.png)

weired thing - image of loading on graph that get data (but not rendering graph) hides

**Comment by iShift** [comment]
example from dev tool:
![image](https://user-images.githubusercontent.com/426427/33983884-c68e40b6-e0c6-11e7-9e75-5b953c4803e0.png)
In that state graphs not updating, they waiting others...

**Comment by torkelo** [comment]
strange, unable to replicate and no other similar reports. Maybe something else has changed?

**Comment by iShift** [comment]
No, how can i prove my words? video?

**Comment by torkelo** [comment]
yes, also do a chrome dev tools performance record, maybe one of your queries return many many thousands of data points and hang your browser for a while so it looks like it is sync

**Comment by iShift** [comment]
fix my problem:
I use datasource ClickHouse - i switch from get to post and add HTTP/2 with nginx
